docs/data/building_permits.md

```diff
@@ -0,0 +1,14 @@
+---
```

Comment:
it looks like all your files have 3 lines of `---` instead of just 2 (opening and closing)